pypif_sdk/interop/mdf.py

```diff
@@ -228,13 +228,17 @@ def _extract_key_value(obj):
     # Parse a Value object, which includes Properties
     if isinstance(obj, Value):
         key = _construct_new_key(obj.name, obj.units)
-        value = None
-        if obj.scalars and len(obj.scalars) == 1:
-            value = obj.scalars[0].value
-        elif obj.scalars:
-            value =  [x.value for x in obj.scalars]
+        value = []
+        if obj.scalars:
+            value = [(val.value if isinstance(val, Scalar) else val)
```

Comment:
When would `val` not be an instance of `Scalar`?

Reply:
The `csv_template_ingester` can return pifs that contain strings in `Value.scalars`. If that's not supposed to be possible, perhaps I've misunderstood where this bug is.